destination/destination.go

```diff
@@ -69,6 +71,7 @@ func (d *Destination) Open(ctx context.Context) error {
 
 	writer, err := sheets.NewWriter(
 		ctx,
+		// haris: when setting d.config in Open() above, we don't set Client. Should we?
```

Comment:
Done.

Reply:
Hmm, I can't see the client being set in Open(), [link](https://github.com/gopherslab/conduit-connector-google-sheets/pull/3/files#diff-7fd240e53ce4c101a889038fcb325fdd36d10e18e771eaa78fdbbf8ea8c700a3R69-R88).

Reply:
It was moved to sheets package, under `NewWriter`

Reply:
The client init functionality has been moved to the sheets package, It is no longer being stored in the config and since we now have a separate package for sheets, our destination doesn't need to be aware of the presence of the http.Client used to interact with the sheets API.